NuCivic/dkan
# Remove css height on recline-nvd3 CIVIC 6001

issue: CIVIC-6001

## Description
Creating a chart with a large number of items on the y axis gets squished looking as the chart has a set height of 480px. To allow the user to adjust the height we need a form field on the chart config to capture this setting.

## Steps to reproduce
1. Create a horizontal bar chart with a large number of items.
2. Confirm that the left side labels are compacted and hard to read.

![screenshot 2017-03-14 16 21 30](https://cloud.githubusercontent.com/assets/314172/24428635/b2edfe44-13d4-11e7-8a6e-792788bdbf9e.png)

## QA Steps
1. Create a horizontal bar chart with the following file. https://dl.dropboxusercontent.com/u/6085083/long-label-test.csv
2. Confirm that the left side labels are compacted and hard to read.
3. Edit the chart and enter a height value such as 850
4. Confirm that the labels now have some breathing room
5. View the chart with IE, and confirm the chart also displays in that browser

## Related PRs/ Merge process
1. merge https://github.com/NuCivic/recline.view.nvd3.js/pull/36
2. close https://github.com/NuCivic/visualization_entity/pull/61
4. update this PR >> edit **drupal-org.make** to set visualization_entity to tag 7.x-1.1
5. merge this PR
